Lib/test/test_fuzz.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+import _fuzz
```

Comment:
This hard dependency on the _fuzz module suggests that this file simply won't work unless _fuzz is available, and AFAICT it won't be available when compiling with gcc or MSVC++.  You need to make sure these tests are skipped when the _fuzz module is unavailable.